content/learn/vp_agof.adoc

```diff
@@ -0,0 +1,375 @@
+
+This is the easiest method if you already use AWS, as it automates the provisioning of resources, including VMs and network configurations. This requires AWS infrastructure and credentials.
+
+=== Custom Ansible controller (API install)
+
+In this method, you provide an existing Ansible Automation Platform (AAP) Controller endpoint, either on bare metal or in a private cloud, without needing AWS or pre-configured VMs. You specify the manifest, endpoint hostname, admin credentials, and pass the installation process to a predefined `controller_config_dir`. This is suitable for complex or custom topologies where you want full control over the deployment.
+
+This method provides maximum flexibility and is designed for advanced users who have their own AAP installations, either on-prem or in complex environments that do not fit into the default or AWS-centric model. You need an existing AAP controller, which might not be ideal for users new to AAP or those looking for more hands-off installation
+
+== Creating a validated pattern using the AGOF framework
+
+Whichever method you use to deploy a validated pattern you need to:
+
+. Clone the link:https://github.com/mhjacks/agof_demo_config[Ansible GitOps Framework Minimal Demo] repository. This is a minimal pattern that demonstrates how to use the Ansible GitOps Framework.
++
+[source,terminal]
+----
+$ git clone https://github.com/validatedpatterns-demos/agof_minimal_demo.git
+----
+
+. Clone the link:https://github.com/validatedpatterns/agof[AGOF] repository. This serves as the provisioner for the pattern
++
+[source,terminal]
+----
+$ git clone https://github.com/validatedpatterns-demos/agof_minimal_demo.git
+----
+
+=== Deploying using the AWS-based install method
+
+This method: 
+
+. Builds an AWS image using Red Hat's ImageBuilder.
+
+. Deploys that image onto a new AWS VPC and subnet.
+
+. Deploys AAP on that image using the command line installer. 
+
+. Hands over the configuration of the AAP installation to the specified `controller_config_dir`. 
+
+.Prerequisites
+
+You need to provide some key information to a file named `agof_vault.yml` created in your home directory. The key pieces of information needed are: 
+
+* The name of a hosted zone created under Route 53 in AWS. For example this could be `aws.validatedpatterns.io`.
+
+* Your AWS account `Account ID`
+
+* Your `aws key`
+
+* Your `secret access key`
+
+.Procedure
+
+. Copy the `agof_vault_template.yml` from the cloned `agof` directory to your home directory and rename it `agof_vault.yml`. 
++
+[source,terminal]
+----
+$ cp ~/agof/agof_vault_template.yml ~/agof_vault.yml
+----
++
+.Agof vault settings
```

Comment:
Suggestion to add some context before starting the table such as 
point 2. Update the `agof_vault.yml` file with the required AWS and Red Hat credentials and configuration settings: